vuepress/docs/v6.3.2/tutorials/solution/pda-tutorial.md

```diff
@@ -52,8 +52,10 @@ Follow the steps below to configure service permissions and connections.
 1. [Login to your Keycloak instance](../../docs/consume/identity-management.md#logging-into-your-keycloak-instance) as an admin.
 
 2. [Assign client roles](../../docs/consume/identity-management.md#authorization) to authorize permissions.
-- Select `pn-b0ceabd7-efbd66b6-entando-pda-plugin-server` from `Client Roles`.
-- Select all roles from `Available Roles`.
+- Select `pn-efbd66b6-b0ceabd7-entando-pda-plugin-server` from `Client Roles`.
```

Comment:
Sorry, I missed this on the last review. This is the 7.0 client id. We need to switch back to the 6.3.2 client id for this tutorial.

Reply:
entando-pda-plugin-server